README.md

```diff
@@ -38,6 +38,10 @@ For reference, here is the wiring diagram to hook up the LED and servo to your R
 > For the servo, note that the red (middle) wire is 5v, the brown wire is ground, and the orange wire is data.
 
 # Bring TJBot to Life
+First, make sure you have configured your Raspberry Pi for TJBot.
+Just run that command to download and install TJBot:
+    $ curl -sL https://raw.githubusercontent.com/ibmtjbot/tjbot/master/bootstrap/bootstrap.sh | sudo -E bash -
```

Comment:
Removing the $ at the start of the line would make double clicking the line for a cut and paste operation easier. Try it [here](https://github.com/ibmtjbot/tjbot/blob/feature/bb-bootstrap-initial/bootstrap/README.md#tjbot-bootstrap)

Removing the $ would also be more consistent with other command in the README.